lib/spack/docs/module_file_support.rst

```diff
@@ -599,7 +599,7 @@ do so by using the environment blacklist:
          all:
            filter:
              # Exclude changes to any of these variables
-             environment_blacklist: ['CPATH', 'LIBRARY_PATH']
+             exclude_environment: ['CPATH', 'LIBRARY_PATH']
```

Comment:
or `exclude_variables`?

Reply:
I like it better -- would `exclude_env_vars` be even more clear?

Reply:
Committed again with `exclude_env_vars`